File: Modul-3/Soal-2/PRAK302_2410817310012_FAISALTANJUNG.py
bilangan = int(input())

if bilangan >= 80:
    print("A")
elif bilangan >= 70 and bilangan <= 79:
    print("B")
elif bilangan >= 60 and bilangan <= 69:
    print("C")
elif bilangan >= 50 and bilangan <= 59:
    print("D")
elif bilangan < 50:
    print("E")

File: Modul-3/Soal-2/PRAK302_2410817310012_FAISALTANJUNG.c
#include <stdio.h>
int main(){
    int bilangan;
    printf("");
    scanf("%d", &bilangan);

    if( bilangan >= 80){
        printf("A\n");
    }
    else if(bilangan >= 70 && bilangan <= 79){
        printf("B\n");
    }
    else if(bilangan >= 60 && bilangan <= 69){
        printf("C\n");
    }
    else if(bilangan >= 50 && bilangan <= 59){
        printf("D\n");
    }
    else if (bilangan < 50){
        printf("E\n");
    }
}